templates/partial/_careers_navigation.html

```diff
@@ -0,0 +1,334 @@
+<div class="dropdown-window-overlay fade-animation"></div>
+<div class="dropdown-window slide-animation">
+	<div class="u-hide p-section" id="careers-menu" aria-hidden="true">
+		<div class="u-fixed-width p-strip">
+			<h2><a class="p-link is-dark" href="/careers">Careers <br class="u-hide--medium u-hide--large" />overview</a></h2>
+		</div>
+		<div class="row">
```

Comment:
This row is without a strip

Reply:
That doesn't seem to work? p-strip still can't change the `span` color property